framework/src/timesteppers/IterationAdaptiveDT.C

```diff
@@ -318,79 +333,80 @@ void
 IterationAdaptiveDT::limitDTToPostprocessorValue(Real & limitedDT) const
 {
   if (_pps_value && _t_step > 1)
-  {
```

Comment:
I know that our style guide says to not have curly braces around one-liners, but in cases like this where there's a nested multi-line block, it seems to me like including the curly braces improves readability.